diff --git a/node_modules/@actions/github/lib/github.d.ts b/node_modules/@actions/github/lib/github.d.ts
index 08f6c48..fb95442 100644
--- a/node_modules/@actions/github/lib/github.d.ts
+++ b/node_modules/@actions/github/lib/github.d.ts
@@ -1,5 +1,5 @@
 import { graphql as GraphQL } from '@octokit/graphql/dist-types/types';
-import Octokit from '@octokit/rest';
+import {Octokit} from '@octokit/rest';
 import * as Context from './context';
 export declare const context: Context.Context;
 export declare class GitHub extends Octokit {
